### PR TITLE
Context menu action array

### PR DESCRIPTION
**Pull request by VictorDeBlas**
#105

### CODE REVIEW

File: src/app/systelab-components/grid/options/grid-options-dialog.component.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<systelab-dialog-header [withInfo]="false" (close)="close();">Options</systelab-dialog-header>
+<systelab-dialog-header [withInfo]="false" (close)="close();">{{ 'COMMON_OPTIONS' | translate | async }}
```
Comment (carlosra85) [inline src/app/systelab-components/grid/options/grid-options-dialog.component.html]: Use Options, there are no translations.

File: src/app/systelab-components/grid/contextmenu/grid-context-menu-cell-renderer.component.ts [hunk 2]
```diff
@@ -25,7 +25,12 @@ export class GridContextMenuCellRendererComponent<T> implements AgRendererCompon
 	}
 
 	public dotsClicked(event: MouseEvent) {
-		this.container.dotsClicked(this.rowIndex, this.data, event);
+		let selectedRows: T | Array<T> = this.data;
+		if ( event.ctrlKey ) {
```
Comment (carlosra85) [inline src/app/systelab-components/grid/contextmenu/grid-context-menu-cell-renderer.component.ts]: Bad formatting?

File: src/app/systelab-components/grid/abstract-grid.component.ts [hunk 3]
```diff
@@ -229,12 +229,15 @@ export abstract class AbstractGrid<T> implements OnInit {
 		const rowId: number = Number(elementId.substr(elementId.indexOf('row')).replace('row', ''));
 		const data: T = this.gridOptions.api.getModel()
 			.getRow(rowId).data;
+		const rowSelecteds: Array<T> = this.gridOptions.api.getSelectedRows();
 
 		if (option && option.action !== null && option.action !== undefined && data !== undefined) {
-			const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
+			// const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
```
Comment (carlosra85) [inline src/app/systelab-components/grid/abstract-grid.component.ts]: Remove commented code

File: src/app/systelab-components/grid/abstract-grid.component.ts [hunk 4]
```diff
@@ -229,12 +229,15 @@ export abstract class AbstractGrid<T> implements OnInit {
...
+		const rowSelecteds: Array<T> = this.gridOptions.api.getSelectedRows();
 
 		if (option && option.action !== null && option.action !== undefined && data !== undefined) {
-			const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
+			// const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
+			const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions, rowSelecteds);
 			option.action(actionData);
 		} else {
-			const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
+			// const actionData: GridContextMenuActionData<T> = new GridContextMenuActionData('' + rowId, actionId, data, this.gridOptions);
```
Comment (carlosra85) [inline src/app/systelab-components/grid/abstract-grid.component.ts]: Remove commented code

File: src/app/systelab-components/grid/abstract-grid.component.ts (hunk 3 above)
Comment (VictorDeBlas) [reply src/app/systelab-components/grid/abstract-grid.component.ts]: Done

File: src/app/systelab-components/grid/abstract-grid.component.ts (hunk 4 above)
Comment (VictorDeBlas) [reply src/app/systelab-components/grid/abstract-grid.component.ts]: Done

**Comment by VictorDeBlas** [comment]
@carlosra85 solved conflicts, please, check it if you can

**Pull request merged**